config/locales/en.yml

```diff
@@ -523,13 +523,20 @@ en:
   shf_documents:
 
     new_shf_document: &new_shf_document New SHF Document
+    new_shf_minutes:  &new_shf_minutes Add Meeting Minutes Document
     edit_shf_document: Edit the SHF document details
     view_shf_document: View the SHF document details
     all_shf_documents: All SHF Documents
+    all_shf_minutes: All Meeting Minutes
+    edit_shf_document_contents: Edit the SHF document contents
```

Comment:
I agree.  I removed "SHF". 

This is used for the button text that appears in the upper-right hand corner when viewing a "static" member page.  (see 2nd screenshot above)